FrontEnd/src/components/SignUp/components/signup/signup-form/RulesModal.module.css

```diff
@@ -0,0 +1,26 @@
+.rules-modal {
+  position: absolute;
+  top: 60%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 500px;
+  max-height: 10vh;
```

Comment:
Is it enough for the modal?

Reply:
We currently have no design and size requirements, so I chose minimal values.

Reply:
I'm not really sure when the design will be provided. I would suggest using some fixed value as 600px. So users with small screens will be able to use it.

Reply:
add  z-index: 100; +   max-height: 600px; +    width: 600px;